test/index-static-singleton.js

```diff
@@ -0,0 +1,20 @@
+var indexStaticWatcher = require('../lib/index-static-watcher').create();
+
+var indexStatic;
+
+exports.get = function() {
+  if (!indexStatic) {
+    throw new Error("indexStatic has not yet been built.");
+  }
+  return indexStatic;
+};
+
+exports.build = function(done) {
+  this.timeout(10000);
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) return done(err);
+
+    indexStatic = newIndexStatic;
+    done();
+  });
+};
```

Comment:
by "test suites" did you mean what's in `app.test.js`?  :eyes:   That file is the only place I see `index-static-singleton.jsx` gets required. ummm